### ISSUE TITLE
Ability to add a path for includes

### ISSUE THREAD

**Issue by savraj** [pwwang/liquidpy#19]
So I'm working on this existing liquid template setup, and in Ruby there's a way to specify an additional directory path to search for includes. Is that possible in liquidpy? 

file structure:

```
_layouts
   default.html
_includes
   header.html
```

inside `default.html` there is an `{% include header.html %}` and have liquidpy find it. Ideally by setting a configuration parameter somewhere to have the path `_includes` searched as well.

**Comment by pwwang** [comment]
Considering adding a `config` tag as well as an option in the construct for some configurations.
```liquid
{% config mode="compact", includes="path/to/include_dir; path/to/another_inc_dir" %}
```
```python
liq = Liquid("""""", liquid_config={"mode": "compact", "includes": ["path1", "path2"]})
```

A temporary solution for your problem is to use an absolute path for each file you need to `include`.

**Comment by savraj** [comment]
cool thanks for the response!

**Comment by pwwang** [comment]
Addressed in `v0.5.0`

https://liquidpy.readthedocs.io/en/latest/extends_include/#directories-to-scan
